provisioning/device/src/polling_state_machine.ts

```diff
@@ -48,18 +48,18 @@ export class  PollingStateMachine extends EventEmitter {
           endSession: (callback) => {
             this._fsm.transition('endingSession', callback);
           },
-          register: (callback, registrationId, authorization, requestBody, forceRegistration) => {
-            this._fsm.transition('sendingRegistrationRequest', callback, registrationId, authorization, requestBody, forceRegistration);
+          register: (callback, request, authorization, requestBody) => {
+            this._fsm.transition('sendingRegistrationRequest', callback, request, authorization, requestBody);
           },
         },
         sendingRegistrationRequest: {
-          _onEnter: (callback, registrationId, authorization, requestBody, forceRegistration) => {
+          _onEnter: (callback, request, authorization, requestBody) => {
             /* Codes_SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_18_012: [ `register` shall call `PollingTransportHandlers.registrationRequest`. ] */
             this._currentOperationCallback = callback;
-            this._transport.registrationRequest(registrationId, authorization, requestBody, forceRegistration, (err, body, result, pollingInterval) => {
+            this._transport.registrationRequest(request, authorization, requestBody, (err, body, result, pollingInterval) => {
```

Comment:
>authorization [](start = 57, length = 13)

I don't know that we need to carry the auth parameter around everywhere. feels like a transport class responsibility to know when to use it. here we're just passing it around without really using it. would also help with the form of the API for AMQP and MQTT I believe.